program/src/processor.rs

```diff
@@ -5937,6 +5944,28 @@ impl Processor {
         }
     }
 
+    #[inline(never)]
+    /// Change the maximum number of MangoAccounts.v1 allowed
+    fn change_fill_incentive_scaling(
+        program_id: &Pubkey,
+        accounts: &[AccountInfo],
+        lm_fill_qty_shift: u8,
+    ) -> MangoResult {
+        const NUM_FIXED: usize = 2;
+        let accounts = array_ref![accounts, 0, NUM_FIXED];
+        let [
+            mango_group_ai, // write
+            admin_ai        // read, signer
+        ] = accounts;
+
+        let mut mango_group = MangoGroup::load_mut_checked(mango_group_ai, program_id)?;
+        check!(admin_ai.is_signer, MangoErrorCode::SignerNecessary)?;
+        check_eq!(admin_ai.key, &mango_group.admin, MangoErrorCode::InvalidAdminKey)?;
+
+        mango_group.lm_fill_qty_shift = lm_fill_qty_shift;
```

Comment:
should print the before and after for governance instructions so it's easier to see on gov ui what is changing and how